scripts/resetdb.sh

```diff
@@ -1,12 +1,58 @@
 #!/bin/bash
-echo "Completely wipe and reset database 'test'."
-read -p "Are you sure? " -n 1 -r
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
-    # A command line supplied -u will override the first argument.
-    mysql -u root "$@" -e 'DROP DATABASE IF EXISTS test;'
-    mysql -u root "$@" -e 'CREATE DATABASE test;'
-    mysql -u root "$@" -e "GRANT ALL ON test.* TO 'test'@'localhost' IDENTIFIED BY 'zaphod';"
-    mysql -u root "$@" -D test < ${GOPATH}/src/github.com/google/trillian/storage/mysql/storage.sql
-fi
-echo
+
+set -e
+
+usage() {
+  echo "$0 [--force] [--verbose] ..."
+  echo "accepts envars:"
+  echo " - DB_NAME"
+  echo " - DB_USER"
+  echo " - DB_PASSWORD"
+}
+
+collect_vars() {
+  # set unset envars to defaults
+  [ -z ${DB_USER+x} ] && DB_USER="root"
+  [ -z ${DB_NAME+x} ] && DB_NAME="test"
+  # format reused supplied envas
```

Comment:
environment variables. i'm so used to calling them envars in other places i never considered writing the whole word :) 

(just fixed)